functions.php

```
@@ -436,6 +454,44 @@ function zw_rest_api_init()
     ]);
 }
 
+/**
+ * Serves the album art of an Aeron track.
+ *
+ * The response body is raw image data; zw_rest_serve_fm_track_image() writes it out instead of
+ * the usual JSON encoding.
+ */
+function zw_rest_fm_track_image(WP_REST_Request $request)
+{
+    $image = \Streekomroep\AeronToolbox::trackImage((string)$request['id']);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Streekomroep\AeronToolbox' in method 'zw_rest_fm_track_image'. (undefined)

(StaticAccess)